.github/workflows/soundness.yml

```diff
@@ -143,7 +143,7 @@ jobs:
     if: ${{ inputs.format_check_enabled }}
     runs-on: ubuntu-latest
     container:
-      image: swiftlang/swift:nightly-6.0-jammy
+      image: swift:6.0-jammy
```

Comment:
There are noble images as well.
I know I filed https://github.com/swiftlang/swift/issues/76555 which mentions 40-50% build regression in _our_ repo, but it should be fine for light workflows.